arrows/vtk/applets/color_mesh.cxx

```diff
@@ -480,14 +474,9 @@ class color_mesh::priv : public kwiver::arrows::vtk::mesh_coloration
       }
     }
 
-    kwiver::arrows::pdal::pointcloud_io pc_io;
-    pc_io.set_local_geo_cs(lgcs);
-    pc_io.save(output_path, points, colors);
-#else
-    throw vital::file_write_exception(output_path,
-                                      "KWIVER was not compiled with PDAL, "
-                                      "cannot write LAS.");
-#endif
+    auto pc_io = vital::algo::pointcloud_io::create("pdal");
+    pc_io->set_local_geo_cs(lgcs);
```

Comment:
We need a check here that `pc_io` was created successfully and is not null before we try to use it.  You can thrown an exception if `pc_io` is null.